todoapp/app/src/test/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailViewModelTest.java

```diff
@@ -179,4 +184,21 @@ private void setupViewModelRepositoryCallback() {
 
         mGetTaskCallbackCaptor.getValue().onTaskLoaded(mTask); // Trigger callback
     }
+
+    @Test
+    public void updateSnackbar_nullValue() {
```

Comment:
remove this new line
add more comments
add a new line between the act and assert
(same for the other test)